vortex-array/src/stats/statsset.rs

```diff
@@ -72,7 +73,7 @@ impl StatsSet {
     }
 
     pub fn constant(scalar: &Scalar, length: usize) -> Self {
-        let mut stats = Self::default();
+        let mut stats = Self::empty();
```

Comment:
this is with_capacity

Reply:
I got rid of `with_capacity` instead, since the first time you call `set` we allocate the full vec of Stat::CARDINALITY